src/index.ts

```diff
@@ -5,3 +5,4 @@ export { DATA_KEYS } from './config/keys';
 export { API_ROUTES } from './api/routes';
 
 export * from './types';
+export { MockWebSocket } from './ws';
```

Comment:
Not sure if we want to export from the index. With some luck the module bundler may even skip it from the production version. We can enforce full path import for testings purposes.